### ISSUE TITLE
Mobile: Look up another teams TV

### ISSUE THREAD

**Issue by rythos42** [TheNAF/naflm#70]
As part of starting a match, I need to know my opponents TV and sometimes I'd like to know before I meet up with them. A feature to look up just their TV would be helpful.

Optional feature could turn this into a more full-features team viewer, but that might be for another Issue.

**Comment by thefloppy1** [comment]
If you use the Match report that I modified, it has both teams TV on it.

**Comment by rythos42** [comment]
This is for the mobile interface, which uses some of the desktop interface elements, but not all of them.

**Comment by thefloppy1** [comment]
oh. well that sort of should have been evident with the title.  sorry.

**Comment by rythos42** [comment]
I rejigged the matches interface to create some more space in the table and added some SQL to get the TV for both teams in the match. 

It's not great, since the match technically *has* a TV listed, but it's 0'd if the match hasn't been played and I don't want to mess with it.

**Comment by rythos42** [comment]
As a reader could probably tell from the commits, I went through a couple options tonight. I like the last one the most and may keep it, but it wasn't easy to find space in this table and keep it uncluttered.

Had to ditch the match date and "my team" TV (since it's at the top). I think I gained space overall.

My own view for my league reads a little bit like a list of Harry Potter books..."CHOP CHOP vs The Ice Drakes", "CHOP CHOP vs The Jokers of Capua"...

The more I look at it, the more I think I'm closing this issue. It's pretty ok looking.
